ops/testing.py

```diff
@@ -100,6 +123,23 @@ def __init__(
         self._defaults = self._load_config_defaults(config)
         self._update_config(key_values=self._defaults)
 
+        # TODO: If/when we decide to allow breaking changes for a release,
+        # change SIMULATE_CAN_CONNECT default value to True and remove the
+        # warning message below.  This warning was added 2022-03-22
+        if not SIMULATE_CAN_CONNECT:
+            warnings.warn(
+                'Please set ops.testing.SIMULATE_CAN_CONNECT=True.'
+                'See https://juju.is/docs/sdk/testing#heading--simulate-can-connect for details.')
+
+    def set_can_connect(self, container: typing.Union[str, model.Container], val: bool):
+        """Change the simulated can_connect status of a container's underlying pebble client.
+
+        SIMULATE_CAN_CONNECT must be True before using this.
```

Comment:
Must SIMULATE_CAN_CONNECT be set to True before calling this method, or before instantiating Harness? The top docstring seems to suggest the latter.

Reply:
This method will raise if it's not set to true when you call it.  *If* you are going to call set SIMULATE_CAN_CONNECT=true, then you should set it before creating the harness.  Clear?

Reply:
I'd say "Calling this method will raise an exception if testing.SIMULATE_CAN_CONNECT is not True."

Reply:
Agree - I pushed up a tweak.